py_ocsf_models/objects/ldap_person.py

```diff
@@ -44,7 +44,7 @@ class LDAPPerson(BaseModel):
     job_title: Optional[str]
     ldap_cn: Optional[str]
     ldap_dn: Optional[str]
-    labels: Optional[list[str]]
+    labels: Optional[dict]
```

Comment:
This will not comply with the OCSF specification https://schema.ocsf.io/1.2.0/objects/ldap_person?extensions=. `labels` must be a String Array.